keyboards/work_louder/micro/info.json

```diff
@@ -11,7 +11,7 @@
         "mousekey": true,
         "nkro": true,
         "encoder": true,
-        "rgblight": true,
+        "rgblight": false,
```

Comment:
This should be reverted.  The board supports both rgb light and rgb matrix. 

If there is additional via config that needs to be added to support that, let me know.

Reply:
Reverted, now it will throw `"Cannot use RGBLIGHT and RGB Matrix using WS2812 at the same time."` and the firmware will be too large.

Reply:
Thanks!

The warning is normal. The work louder boards are using two ws2812 strips on different pins, and using a hack to support them.  Not a great solution, and the compiler complains. 

And I'll take a look at the compiled size.